### PR TITLE
Add docs for SRF column accessors

### LINKED ISSUE #3070: sql: add the ability to access a column from the results of an SRF
PR: https://github.com/cockroachdb/cockroach/pull/24832

From release notes:
> Set Returning Functions (SRF) can now be accessed using `(SRF).x`, where `x` is the name of a column returned from the SRF or a `*`. For example, `SELECT (information_schema._pg_expandarray(ARRAY['c', 'b', 'a'])).x` and `SELECT (information_schema._pg_expandarray(ARRAY['c', 'b', 'a'])).*` are now both valid. Also, the naming of the resulting columns from SRFs now provide more information about the resulting tuple. [#24832][#24832] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes #3070

### CODE REVIEW

**Comment by rmloveland** [comment]
Specific change is in the [Table generator functions](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/941128ce8940cfe27c3f90de9d2d72a2be4c1f29/dev/table-expressions.html#table-generator-functions) section.

File: v2.1/table-expressions.md [hunk 1]
```diff
@@ -198,7 +198,27 @@ For example:
...
+
+~~~
+ x | n
+---+---
+ 1 | 1
+ 2 | 1
+(2 rows)
+~~~
+
+{{site.data.alerts.callout_info}}Currently CockroachDB only supports a small set of generator functions compatible with <a href="https://www.postgresql.org/docs/9.6/static/functions-srf.html">the PostgreSQL set-generating functions with the same names</a>.{{site.data.alerts.end}}
```
Comment (knz) [inline v2.1/table-expressions.md]: I understand that callouts can now use markdown syntax, @jseldess was explaining this to me the other day.

Comment (jseldess) [reply v2.1/table-expressions.md]: Yep, in fact @rmloveland was the one to discover this. Just need to put the opening and closing tags on their own lines.

**Comment by rmloveland** [comment]
@jseldess mind giving this one a look again?  thanks!

**Comment by jseldess** [comment]
Oh, also, is this PR fixing an issue? If so, can you use `Fixes #<issue>` or the like?

**Pull request merged**